sagemaker-clarify/computer_vision/image_classification/explainability_image_classification.ipynb

```diff
@@ -438,6 +438,9 @@
     "    * **content_type**: Specifies the type of input expected by the model.\n",
     "* **predictions_config**: Config object to extract a predicted label from the model output.\n",
     "    * **label_headers**: This is the list of all the classes on which the model was trained.\n",
+    "        * **NOTE**: Predictions for Image Classification models MUST come as a list of probability values (of the same size as the list of label_headers). Ensure model output is in this format.\n",
```

Comment:
The notes here are not valid anymore. The latest Clarify container image supports JSON model input for CV image classification explainability.